docs/manual/scala/guide/cluster/UsingAkkaPersistenceTyped.md

```diff
@@ -187,12 +187,12 @@ Akka supports both [programmatic passivation](https://doc.akka.io/docs/akka/2.6/
 
 The messages (commands, replies) and the durable classes (events, state snapshots) need to be serializable to be sent over the wire across the cluster or be stored on the database. Akka recommends [Jackson-based serializers](https://doc.akka.io/docs/akka/2.6/serialization-jackson.html?language=Scala) --preferably JSON, but CBOR is also supported-- as a good default in most cases. On top of Akka serializers, Lagom makes it easy to add [[Play-JSON serialization|SerializationPlayJson]] support, which may be more familiar to some Scala developers.
 
-In [Akka Persistence Typed](https://doc.akka.io/docs/akka/2.6/typed/persistence.html?language=Scala), in particular, and when you adopt CQRS/ES practices, commands will include a `replyTo: ActorRef[Reply]` field. This `replyTo` field will be used on your code to send a `Reply` back, as shown in the examples above. Serializing an `ActorRef[T]` requires using the Akka Jackson serializer, meaning you cannot use Play-JSON to serialize commands.
```

Comment:
@leviramsey while it's possible to serialize an `ActorRef` using play-JSON in theory, the Lagom use case ia a bit more intricate. IIRC, the problem lies in the ordering in which `SerializerRegistry` and other DI components initialize. That ordering makes it hard (if not impossible) to inject the `ActorRefResolver` into the play JSON formatters.
If you have a working example to contribute to this documentation it'd be a great addition.

Reply:
I had forgotten Lagom's DI infrastructure (it's been a few years since I really worked with lagom).  Yeah I can see how it would be difficult to sequence the initialization so this is after `ActorSystem` startup.

I'll close this PR because I don't have the context to rig up an example.